content/docs/atom-guide/look-dev/six-point-lighting-tutorial.md

```diff
@@ -0,0 +1,439 @@
+
+   ```glsl
+   float3 GetIblDiffuse(Surface surface, float3 diffuseResponse)
+   {
+      float3 rightSample = GetIblSample(surface, float3(1.0f, 0.0f, 0.0f));
+      float3 leftSample = GetIblSample(surface, float3(-1.0f, 0.0f, 0.0f));
+      float3 topSample = GetIblSample(surface, float3(0.0f, -1.0f, 0.0f));
+      float3 bottomSample = GetIblSample(surface, float3(0.0f, 1.0f, 0.0f));
+      float3 frontsideSample = GetIblSample(surface, float3(0.0f, 0.0f, 1.0f));
+      float3 backsideSample = GetIblSample(surface, float3(0.0f, 0.0f, -1.0f));
+
+      float3 totalDiffuseSample = (leftSample * surface.left) 
+                                 + (rightSample * surface.right) 
+                                 + (topSample * surface.top) 
+                                 + (bottomSample * surface.bottom) 
+                                 + (frontsideSample * surface.frontside) 
+                                 + (backsideSample * surface.backside);
+
+      return diffuseResponse * surface.albedo * totalDiffuseSample;
+   }
+   ```
+
+   {{< tip >}}
+   Multiplying the sampled color by the surface property is the key to making this lighting approximation work. Recall that the surface property gives us the intensity of the light on a texel if light were to come from that respective direction. Therefore, multipling the sampled color by the intensity scales the color value appropriately.
+
+   For example, at the top of a texture, `surface.top` would be very intense (around `255.0`), but `surface.bottom` would be very mild (around `0.0`). Therefore, at the top of the texture the `bottomSample` would essentially have no effect on the color. 
+   {{< /tip >}}
+
+5. Open the Editor and turn off any lights. You should see the colors on your material reflect those of the skybox (blue at the top and orange at the bottom).
+
+   {{< video src="/images/atom-guide/six-point-lighting/ibl.mp4" autoplay="true" loop="true" width="100%" muted="true" info="Video of rotating the six-point lighting material to see how IBL affects the " >}}
+
+6. Turn on the lights again and observe how IBL works with the directional lighting!
+
+   {{< video src="/images/atom-guide/six-point-lighting/final.mp4" autoplay="true" loop="true" width="100%" muted="true" info="Video of changing direction of the light applied onto the six-point lighting cloud." >}}
+
+Awesome, you added custom directional lighting and IBL!
+
+## Download the AtomTutorial Gem sample
+Now that you've completed this tutorial, you can compare your results to our working version of six-point lighting in the **AtomTutorials** Gem in the [o3de/sample-code-gems repository](https://github.com/o3de/sample-code-gems). You can either download and place the [final working six point lighting files]() in your project, or you can [download the Gem and add it to the engine]() if you haven't already in the [vertex deformation tutorial](). 
```

Comment:
For `path`, write out the name of the path to the files. For `link`, provide a link to the files in GitHub.
```suggestion
Now that you've completed this tutorial, you can compare your results to our working version of six-point lighting in the **AtomTutorials** Gem in the [o3de/sample-code-gems repository](https://github.com/o3de/sample-code-gems). You can either download the final six point lighting files from [path](link) and place them in your project, or you can download the Gem and add it to the engine (see [Adding and Removing Gems in a Project](/docs/user-guide/project-config/add-remove-gems/)).
```